gpt3/README.md

```diff
@@ -1,13 +1,19 @@
 # This is an example of GPT3 using MS-AMP
 
 ## Apply patch to Megatron-DeepSpeed
```

Comment:
This title is not changed. `## Apply patch to Megatron-DeepSpeed`

Reply:
Fixed.